data/json/items/comestibles/junkfood.json

```
@@ -1551,5 +1551,53 @@
     "fun": 15,
     "material": [ "veggy", "junk" ],
     "copy-from": "gelatin_dessert_base"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pasta_carbonara",
+    "name": { "str": "pasta carbonara" },
+    "weight": "150 g",
+    "color": "yellow",
+    "spoils_in": "21 days",
+    "container": "pan",
+    "comestible_type": "FOOD",
+    "symbol": "c",
+    "quench": -2,
+    "healthy": -1,
+    "calories": 565,
+    "description": "a pan containing pasta carbonara, a famous Italian dish made with egg, pecorino and bacon.",
```

```suggestion
    "description": "A famous Italian pasta dish made with egg, pecorino and bacon.",
```
Referring to the container won't make sense if the container can be swapped 🤔